cocotb/binary.py

```diff
@@ -279,11 +286,19 @@ def _adjust_twos_comp(self, x):
         return rv
 
     def get_value(self):
-        """value is an integer representation of the underlying vector"""
+        """value is an integer representation of the underlying vector.
+
+        Returns:
+            int: integer representation
```

Comment:
I think this description is a bit redundant.  I think the first part of it should just be clearer.
```python
"""Return the integer representation of the underlying vector."""
```